framework/core/js/src/common/Application.tsx

```diff
@@ -550,7 +550,11 @@ export default class Application {
         break;
 
       default:
-        if (this.requestWasCrossOrigin(error)) {
+        const code = error.response?.errors?.[0]?.code;
+
+        if (code === 'db_error' && app.session.user?.isAdmin()) {
```

Comment:
Should we print the current db driver?

Reply:
The admin is able to see their db driver in the admin page so I don't think it's necessary.